data_structures/src/error.rs

```diff
@@ -357,6 +357,18 @@ pub enum BlockError {
         proof_pkh: PublicKeyHash,
         signature_pkh: PublicKeyHash,
     },
+    /// Value Transfer Weight Overflow
+    #[fail(
+        display = "Value Transfer Transactions weight: {}, exceed the maximum weight: {}",
+        weight, max_weight
+    )]
+    ValueTransferWeightOverflow { weight: u32, max_weight: u32 },
```

Comment:
Is this really overflowing or just hitting the limit? If the latter, isn't it a bit misleading?

Reply:
it is really overflowing